Source/VCGeneration/IConditionGenerationLogger.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+using Microsoft.Boogie;
+
+namespace VC;
+
+public interface IConditionGenerationLogger {
+  public void ReportSplitResult(Split split,
```

Comment:
Consider renaming this to `LogSplitResult` since it's a method in the a `Logger` interface.

Reply:
Renamed instead the interface to `IConditionGenerationPrinter` since it's extended by `OutputPrinter`